libremesh.sdk.config

```diff
@@ -33,6 +32,7 @@ CONFIG_PACKAGE_lime-webui=m
 CONFIG_PACKAGE_bmx7-auto-gw-mode=m
 CONFIG_LIMEWEBUI_ES=y
 CONFIG_PACKAGE_lime-app=m
+CONFIG_PACKAGE_luci=m
```

Comment:
this is outside of the fix.
please remove... apart from that, awesome!

Reply:
...luci was also selected by lime-full (see [here](https://github.com/libremesh/lime-packages/pull/513/files#diff-94b18f1edb9c303e90a01ac03aa32ff6L26)).

Do you confirm that we don't want this to be included in the SDK packages?

Reply:
Not yet I believe. thanks for catching that!